assets/src/web-stories-block/css/lightbox.css

```diff
@@ -1,18 +1,20 @@
 /* Lightbox styles */
+.ws-lightbox-open {
```

Comment:
Let's use the same `.web-stories` prefix as for other class names.

So `.web-stories-lightbox-open`

Reply:
updated to use `.web-stories-` prefix